src/sgame/sg_cmds.cpp

```diff
@@ -3536,6 +3564,13 @@ static void Cmd_Build_f( gentity_t *ent )
 	vec3_t      origin, normal;
 	int         groundEntNum;
 
+	if ( G_admin_permission( ent, ADMF_NO_BUILD ) ) // this is separate from the denybuild check as the
```

Comment:
Admins can already deny building rights without a vote with the `denybuild` command so the message was wrong anyway. The checks should be combined and the message made more generic.

Reply:
This is true, I could submit a separate PR to fix the incorrect messaging instead, or I could fix that as part of this one if you prefer?

Reply:
You can do it this one. Ideally all of the existing commits would be squashed together, and there would be one other commit for the message.

Reply:
Addressed in https://github.com/Unvanquished/Unvanquished/pull/2356/commits/0575a264686cab9ae08de38ab579b696a985ed78

`MN_B_REVOKED` can be adjusted once this is approved.